Library/Formula/keybase.rb

```diff
@@ -1,33 +1,32 @@
 class Keybase < Formula
   desc "Command-line interface to Keybase.io"
   homepage "https://keybase.io/"
-  url "https://github.com/keybase/node-client/archive/v0.8.23.tar.gz"
```

Comment:
As soon as the hombrew PR gets accepted, we are flipping the switch on the website to new instructions.